README.md

```diff
@@ -136,7 +136,7 @@ constructor documentation for more options.
 
 Note that `click` will trigger hover events before clicking. To disable this,
 set the `skipHover` option to `true`. Also note that trying to click an element
-with `user-events` being set to `"none"` (i.e. unclickable) will throw an eror.
+with `user-events` being set to `"none"` (i.e. unclickable) will throw an error.
```

Comment:
```suggestion
with `pointer-events` being set to `"none"` (i.e. unclickable) will throw an error.
```

Thanks for spotting the typo.
While we're at it, let's fix the property name to.